test/test-framework/deploy/crds/cache.example.com_memcachedrs_crd.yaml

```diff
@@ -1,58 +1 @@
-apiVersion: apiextensions.k8s.io/v1beta1
```

Comment:
No changes should need to be made other than import strings if we're using a drop-in replacement.